common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -123,61 +136,49 @@ $sequence--border-color: #C8C8C8;
           }
         }
 
-        //video
-        &.seq_video {
-          &.inactive {
-            background-image: url('../images/sequence-nav/video-icon-normal.png');
-          }
+        &:hover, &:focus {
+          background-color: $white;
+          background-repeat: no-repeat;
+          background-position: center 14px;
 
-          &.visited {
-            background-image: url('../images/sequence-nav/video-icon-visited.png');
+          .icon {
+            color: rgb(80, 80, 80);
           }
+        }
+
+        //video
+        &.seq_video {
 
-          &.active {
-            @extend .active;
-            background-image: url('../images/sequence-nav/video-icon-current.png');
+          .icon:before {
+            content: "\f03d";
```

Comment:
NIT: please include the Font Awesome icon name in a comment next to this for easy reference.